### ISSUE TITLE
Update status of Variable fonts on Firefox

### ISSUE THREAD

**Issue by ngdangtu-vn** [Fyrd/caniuse#6625]
I'm currently using the FF 108 on Linux KDE Plasma distro and I can say it does support variable fonts. But I didn't see any report for it on CIU so I thought I would fire an issue about this note.

**Comment by Ugoku** [comment]
Caniuse already has a listing for variable fonts: https://caniuse.com/variable-fonts and it lists FF as supported

**Comment by ngdangtu-vn** [comment]
> FF as supported

That is for Mac & ~~Cheese~~ Windows. Anyway, let me rephrase my words. I think CIU should update more detail on how FF handle VF in Linux like what lib it requires or which distros support it. My issue focus on Linux rather than other 2 OS.
